ideas.md

```diff
@@ -0,0 +1,16 @@
+# 💡 Actions Idea List
+
+This is a list of ideas of actions and workflows that people would like to see one day.
+
+If you would like to contribute to this, feel free to add to the list:
```

Comment:
Maybe a note in here that they should first check if it exists before adding to the list

Reply:
Done